CHANGELOG.md

```diff
@@ -15,7 +17,8 @@
 
 ### Internal
 
-* Factor out internal interface ManagedObject
+* Factor out internal interface ManagedObject.
+* Use Object Store to do table initialization.
```

Comment:
If detailed message appears as java exception message instead of the "expected field count 15 but was 14" then that is super-nice